### DISCUSSION TITLE
Migrate from XML to Jetpack Compose or Kotlin Multiplatform

### DISCUSSION THREAD

**Discussion by KhubaibKhan4** [libre-tube/LibreTube#5991] [Ideas]
### Describe your suggested feature

I think, you guys are getting some outdated bugs. I think, Its better to move this project from xml to Jetpack Compose and will be more greater, If move this to Kotlin or Compose Multiplatform. Surely, I would love to do that. If you interested, Let's discuss

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
I'm personally not interested in working on a Jetpack Compose migration - the whole UI would need to be re-written.

MDC Android is and will stay supported, so there's no pressure to migrate.

The migration would be a long progress and the source of many new incoming bugs, and I don't have time and energy for that. I see that you're willing to help, but I don't really trust people to rewrite the whole code that never actively contributed to the project before.
